### DISCUSSION TITLE
Bricked (?) CarlinKit 5.0 CPC200-2air / U2AC, no LED status

### DISCUSSION THREAD

**Discussion by dough29** [ludwig-v/wireless-carplay-dongle-reverse-engineering#184] [Q&A]
Hello,

CarlinKit 5.0 CPC200-2air / U2AC appears bricked after trying to update.

When I plug it into the wall, I barely see a green light blinking near the USB A port, but nothing else.

I managed to dump the MX25L12835F chip and I think I need a working one to unlock it.

Can anyone provide help on this?

Thanks.

**Comment by dough29** [comment]
I tried the BIOS file [MX25L12835F_FW_20210306.bin](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/blob/master/Flash_Dump/MX25L12835F_FW_20210306.bin) from @bronken

Now the device still doesn't turn on the LED on the CarlinKit logo but near the USB A port there is a solid red LED and when I plug in a USB stick there is some activity.

But it doesn't seem to apply the update file that's on the USB stick, nothing happens anymore.

**Comment by dough29** [comment]
Tried the _modified.bin_ file from https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/59#discussioncomment-1679085

When I plug the USB stick it writes a file _userspace.log_

So it looks like the device is able to start, just needs the right flash image 😎

**Reply by Levonson01** [reply]
Hi, did you manage to flash the right image? I saw you posed an updated firmware here: [https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/tree/f8ceb33c742fd40a2497e4ce7ef73d94ac2f0c33/Firmware/U2AC](url). Can you please describe in detail how you flashed the MX25L12835F :). Did you used a rasperry pi or a pc. Thanks a lot! :D

**Reply by Levonson01** [reply]
Hey, I managed to flash the modified.bin with my CH341 Programmer. But now I also got the red LED :/

**Reply by dough29** [reply]
@Levonson01 hello !

So yes I flashed that dump from @bronken with also a CH341 just to check something else happens : it the case.

Now I don't have to dump for the CarlinKit 5.0 CPC200-2air / U2AC.

I just managed to download the firmware before I did the update, I then failed but I had the file to share if it can help.

For now my device is still bricked.

**Reply by Levonson01** [reply]
So do you think we just need a suitable BIOS for our Carlinkit 5.0, but for now no one has it publicly released? :/

**Reply by dough29** [reply]
I think we should find someone with a working device to dump the MX25L12835F chip so we can restore it to our bricked devices to get them back alive

**Reply by Levonson01** [reply]
Alright, if you manage to fix this could you please inform me here? Thanks

**Comment by dough29** [comment]
Hello,

I tried reading a brand new CarlinKit 5.0 CPC200-2air / U2AC MX25 chip (never plugged) and flashing it to the broken one : no luck 😢

**Reply by Levonson01** [reply]
I guess there is no chance reviving that thing 🥲

**Reply by mival** [reply]
@dough29 hi, do you have firmware binary from working unit? I also bricked V5

**Reply by Debugx00** [reply]
@dough29 upload *.bin dump from working device pls. I have copy my own *.bin dump. But i have problem, web-interface working, red ui (with carplay choose menu) not-working seems like carlinkit not connected to headunit. And led status pink, usb with firmware didn't help 🫠

**Reply by mival** [reply]
@Debugx00 Can you please upload you bin. I have totally dead device

**Reply by Debugx00** [reply]
> @Debugx00 Can you please upload you bin. I have totally dead device

Yep, look here [My GitHub discussion](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/197)
